hare/messagevalidation.go

```diff
@@ -17,9 +17,37 @@ func NewMessageValidator(signing Signing, threshold int, defaultSize int, valida
 	return &MessageValidator{signing, threshold, defaultSize, validator}
 }
 
+func (validator *MessageValidator) SyntacticallyValidateMessage(m *pb.HareMessage) bool {
```

Comment:
Should rename this function (confusing since we already have a syntax validation func). This one actually also cvalidate the sig.

Reply:
validating the sig is also part of the syntactical validation. I renamed the private method to `isStructureValid`